channels/fast-4.3.yaml

```diff
@@ -1,4 +1,7 @@
 name: fast-4.3
+feeder:
+  name: candidate-4.3
```

Comment:
how are we handling n-1 minor feeders? both candidate-4.2 and candidate-4.3 feed into fast-4.3 (after we open the minor version upgrades for that release)

Reply:
> how are we handling n-1 minor feeders? both candidate-4.2 and candidate-4.3 feed into fast-4.3 (after we open the minor version upgrades for that release)

I think only candidate-4.3 feeds fast-4.3.  It's just that we add both 4.2 and 4.3 releases to candidate-4.3.  There is a potential use-case for having multiple feeders, but I think it's all of the candidate-4.* channels feeding the version-agnostic `candidate` channel from the proposal in #112.  And that's going to be a bit tricky because of the possibility of implicit edge blocking as discussed in #112, so I haven't dipped into that here.  Can we punt on it and revisit if/when we get around to considering #112?  Or should we hash it out in this PR?